python_modules/dagster-webserver/dagster_webserver/webserver.py

```diff
@@ -231,15 +230,31 @@ def index_html_endpoint(self, request: Request):
                 """)
 
     def build_static_routes(self):
-        def next_file_response(file_path):
-            with open(file_path, encoding="utf8") as f:
-                content = f.read().replace(
-                    "BUILDTIME_ASSETPREFIX_REPLACE_ME", f"{self._app_path_prefix}"
-                )
-                return Response(content=content, media_type=guess_type(file_path)[0])
+        temp_dir = tempfile.TemporaryDirectory() if self._uses_app_path_prefix else None
```

Comment:
this approach is still rough since it will cause thrash if you hit different web server replicas or any time a server process restarts

Reply:
Yeah agreed. I could just build it directly into the real directory but the logic will be a bit more complicated in case you change the path-prefix between launches

Reply:
most actually deployed oss webservers will be in containers where we cant realistically persist written files